AMaDda/Screens/ProfileModal/Controllers/ProfileModalViewController.swift

```diff
@@ -6,24 +6,48 @@
 //
 
 import UIKit
-import SwiftUI
+
+// TODO: edit, add viewController에서 delegate 구현 필요
+protocol profileModalViewDelegate: AnyObject {
```

Comment:
Protocol 이름도 large camel case를 적용한답니당
```suggestion
protocol ProfileModalViewDelegate: AnyObject {
```

Reply:
앗 .. 이런 실수를... 반영하겠습니다 ㅎㅎ

Reply:
반영 완료하였습니다 ~ d2bdce2f428cb1b4a9cfee62512c9db335831569